schema/place.js

```diff
@@ -6,6 +6,39 @@ class Place extends BaseModel {
     return 'places';
   }
 
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [],
```

Comment:
We probably want to add some required fields here, and also a new migration to update the DB. I would suggest `site` `name` `holding` and `suitability` as per the form validation. Thoughts?

Reply:
My preference would be to do this work in a separate ticket to isolate the changes?

Reply:
I'd like to add the required properties here anyway, even if they're not _yet_ in the migrations. Because they are actually required.